pystruct/learners/ssvm.py

```diff
@@ -18,6 +23,15 @@ def __init__(self, model, max_iter=100, C=1.0, verbose=0,
         self.n_jobs = n_jobs
         self.logger = logger
 
+        if self.n_jobs == -1:
+            self._n_jobs = cpu_count()
```

Comment:
Unfortunately we shouldn't do this in the ``__init__`` to follow scikit-learn conventions. Also, with the current implementation I don't understand if joblib.Parallel is ever used. It is not, right?